src/js/components/Pagination/Pagination.js

```diff
@@ -176,9 +178,41 @@ const Pagination = forwardRef(
       ...navProps[control],
     }));
 
+    const itemsBegin = step * (activePage - 1) + 1;
+    const itemsEnd = itemsBegin + step - 1;
+    let showSummary;
+    if (showSummaryProp === true)
+      showSummary = (
+        <Text>
+          Showing {itemsBegin} - {itemsEnd} of {numberItems}
+        </Text>
+      );
+    else showSummary = showSummaryProp;
```

Comment:
In this case, I think summary is a "thing", a noun. The name `showSummary` seems like an instruction - do or do not - and feels like that naming would be more applicable applied to a function or state variable.

```suggestion
    let summary;
    if (showSummaryProp === true)
      summary = (
        <Text>
          Showing {itemsBegin} - {itemsEnd} of {numberItems}
        </Text>
      );
    else summary = showSummaryProp;
```

Reply:
I like `summary` -- I think we should change the prop name to match as well.

Reply:
Changed to `summary`.